consensus/event_loop.go

```diff
@@ -25,6 +25,9 @@ func NewFastHotStuffEventLoop() *fastHotStuffEventLoop {
 // params:
 //   - crankTimerInterval: crank timer interval duration must be > 0
 //   - timeoutBaseDuration: timeout base duration must be > 0
+//   - genesisQC: a quorum certificate at the genesis for the chain. This QC is a trusted input
```

Comment:
Can we make it more clear that this is a QC for the start of the PoS chain? Don't want to get confused w/ genesis block for the whole chain